lib/logstash/inputs/imap.rb

```diff
@@ -193,6 +209,11 @@ def parse_mail(mail)
         end
       end
 
+      # Add attachments
+      if attachments and attachments.length > 0
```

Comment:
We typically prefer to use `&&` over `and` in Boolean expressions (see [this](https://stackoverflow.com/questions/1426826/difference-between-and-and-in-ruby) StackOverflow article for some explanation)

Reply:
Awesome, thanks for the link, very good to know!